iceoryx_posh/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.5)
-set(iceoryx_posh_VERSION 0.16.0.1)
+set(iceoryx_posh_VERSION 0.16.0.3)
```

Comment:
There is already a 0.16.1 release and the version was updated properly. Please set to 0.16.1 @dkroenke Is that ok?